<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>671</y>
     <width>132</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть </string>
   </property>
  </widget>
  <widget class="QLabel" name="outputWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>401</width>
     <height>540</height>
    </rect>
   </property>
   <property name="text">
    <string>Вывод</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>400</width>
     <height>540</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Изображение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recoveryButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>670</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать маску</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>670</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
